.github/workflows/storybook.yml

```
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+jobs:
+  publish:
+    name: Publish Storybook
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)